specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/preview/2018-06-30-preview/guestconfiguration.json

```diff
@@ -369,6 +369,167 @@
           "application/json"
         ]
       }
+    },
+      "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachines/{vmName}/providers/Microsoft.GuestConfiguration/softwareUpdateProfile": {
```

Comment:
if this is a singleton, add a segment to the URL as below 
.../softwareUpdateProfile/{profileName}. Now profileName can be made as an enum with just one possible value which can be called "default". So the name of this singleton would be "default"

Reply:
Done